OpenCO2_Sensor.ino

```diff
@@ -710,13 +736,18 @@ void loop() {
 #ifdef MQTT
     if (!error && !BatteryMode) {
       if (WiFi.status() == WL_CONNECTED) {
-        mqttClient.beginMessage("co2_ppm");
-        mqttClient.print(co2);
-        mqttClient.endMessage();
-        mqttClient.beginMessage("temperature");
+        mqttClient.poll();  // needed?
+        int rc1, rc2, rc3;
+        rc1 = mqttClient.beginMessage("openco2/co2_ppm");
+        rc2 = mqttClient.print(co2);
+        rc3 = mqttClient.endMessage();
+        char result[100];
+        snprintf(result, 100, "MQTT msg sent %d %d %d", rc1, rc2, rc3); // "MQTT msg sent 1 4 0"
```

Comment:
Undocumented return code, maybe:

1 = ok
4 = length (having 4 digits CO2 right now, need to vent :)
0 = fail